packages/client/src/components/GrantsTableNext.vue

```diff
@@ -2,15 +2,15 @@
   <section class="container-fluid" style="margin: 10px;" >
     <b-row class="my-3">
       <div class="ml-3">
-        <SavedSearchPanel />
+        <SavedSearchPanel @edit-filter="openSearchForEdit" @filters-applied="paginateGrants" />
```

Comment:
Saved search panel uses events to control parent actions, i.e. telling it to open the search panel or refetch grants.